examples/Thermostat/slice/Reading.slice

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) ZeroC, Inc.
+
+module Igloo
+
+/// Represents a snapshot of the thermostat state at a point in time.
+struct Reading {
+    /// The point in time where this Reading was taken.
+    timeStamp: WellKnownTypes::TimeStamp
+
+    /// The current temperature (in °F).
+    temperature: float32
+
+    /// The target temperature (in °F).
+    setPoint: float32
```

Comment:
After reading the whole example, I understand, but I was initially confused by these names. I think these would be clearer:
```suggestion
    /// The actual temperature (in °F).
    actual_temperature: float32

    /// The target temperature (in °F).
    target_temperature: float32
```

1) While skimming, I thought "current temperature" was the "current _target_ temperature". Actual temperature better reflects what it's measuring.
`temperature` stores the current actual temperature
`setPoint` stores the current target temperature.

2) Most readers will be unfamiliar with the term 'setPoint', even though we explain it one place. Plus, in our comments, we always say "target temperature", might as well embrace it.
Also, then `changeSetPoint` could be renamed to `setTargetTemperature`.
getters and setters I'm familiar with, but a function named `changeX` sounds... weird...

Reply:
"set point" is the correct term. Target temperature is a way to describe "set point" for people unfamiliar with this term.

Reply:
> Most readers will be unfamiliar with the term 'setPoint', even though we explain it one place

I agree with Austin on this while 'set point' is the more correct term I feel like the example is clearer for people who do not know much about AC and Thermostats with the terms "actual" and "target" temp.

Reply:
I much prefer set point. It's the correct term.

Reply:
I think set point is fine.